### DISCUSSION TITLE
[Feature Request] Output multiple responses with just one request

### DISCUSSION THREAD

**Discussion by capybarahero** [oobabooga/text-generation-webui#2883] [Ideas]
Hi all, 

Would be great if the generator (and its API) could return more than one response similar to what the parameter `n` does on the [OpenAI completions request](https://platform.openai.com/docs/api-reference/completions/create#completions/create-n).

ps.: Thanks for the great stuff you built :rocket:

**Comment by tokenwizard** [comment]
Agreed. Many times the chat gets cut short in mid-sentence and it is nearly impossible to get the AI to continue where it left off. It would be great if it could just send multiple messages so it could complete a long story (just like a human would do to get around character limits).

**Reply by missionfloyd** [reply]
Clicking Continue does exactly this, it continues the last message.

**Reply by TechScorpion** [reply]
When I click "continue" after the chatbot's message it adds "..." for a bit and then removes it and the output generated says "0 tokens" or "1 token". So it's not continuing the last message or story.

It works for continuing a message after you pressed "stop" while it was outputting, but it doesn't seem to for those you didn't.
